iceoryx_binding_c/include/iceoryx_binding_c/types.h

```diff
@@ -79,5 +79,11 @@ struct iox_listener_storage_t_
 };
 typedef struct iox_listener_storage_t_ iox_listener_storage_t;
 
+/// @brief handle of the chunk header
+typedef struct
+{
+    // could be empty but then we get `struct has no members` warning
+    uint8_t do_not_touch_me[1];
```

Comment:
touch

i had to touch it ... i could not resist

Reply:
The name is still bad. It is the data buffer of the object.

Reply:
Yeah but the public variable name suggest to the user that they never ever write something into this thing.

Reply:
yes, it just to have a typed pointer instead of an alias to `void*`. This make the usage a tad safer